src/test/controllers/localStorage_test.ts

```diff
@@ -85,22 +85,34 @@ suite('LocalStorageController', () => {
     });
   });
 
-  suite('with default value', () => {
```

Comment:
sorry last one 🙈  it looks like you overwrote the old `with default value` suite here (which needs to still exist, was already in main)

basically what you have in the sessionStorage tests is right

Reply:
Sorry, copy and paste mistake 🤦